pom.xml

```diff
@@ -6,23 +6,36 @@
 
     <groupId>de.thbingen.info2</groupId>
     <artifactId>HalloTrain</artifactId>
-    <version>1.0-SNAPSHOT</version>
+    <version>2.0-SNAPSHOT</version>
 
     <properties>
 	<project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
 	<kotlin.version>1.1.51</kotlin.version>
+    <main.class>trainSimulation.Main.kt</main.class>
```

Comment:
Du hast den Branch zu diesem PR nicht auf der neusten Version von `master` angefangen und mittlerweile ist in `master` an genau der gleichen Stelle eine Änderung dazugekommen ("zufälligerweise" fast die gleiche Zeile). Beim mergen müsste Git also diese Änderung hier und die auf `master` zusammenführen, aber weil sie in der gleichen Zeile kollidieren geht das nicht automatisch (Git weiß ja nicht, welche du brauchst). Das Resultat ist ein  "Merge Conflict", dne GitHub grau anzeigt.

Reply:
allerdings weiß ich nicht wieso, ich hab den git pull augesführt und hab auch alles auf dem aktuellen Stand gehabt, außer die pom, daher hab ich die aus unserem Master von GitHub kopiert

Reply:
Der letzte commit von `Zweig4`, der auch in `master` ist, ist 5c25f07 vom 27. Oktober. Wenn du da schon angefangen hast an diesem Branch zu arbeiten, hast du eigentlich alles richtig gemacht. Man kann Branches auch ruhig frühzeitig anfangen und dann die neusten Änderungen von `master` reinholen, aber das ist nicht immer ganz einfach.

Reply:
alles klar, habe eine neue Auflage in #38 pull requested. Damit kann ich diesen hier schließen?